packages/@okta/vuepress-site/docs/concepts/auth-overview/index.md

```diff
@@ -7,7 +7,7 @@ meta:
 
 # OAuth 2.0 Overview
 
-This page will give you an overview of OAuth 2.0 and OpenID Connect and their Okta implementations. It will explain the different flows, and help you decide which flow is best for you based on the type of application that you are building. If you already know what kind of flow you want, you can jump directly to:
+This page provides an overview of OAuth 2.0 and OpenID Connect and how these different flows can be implemented using Okta. It will help you decide which flow is best for you based on the type of application that you are building. If you already know what kind of flow you want, you can jump directly to:
 
 - [Implementing OAuth 2.0 Authentication](/docs/concepts/auth-overview/#recommended-flow-by-application-type)
 - [SAML Authentication with OIDC](/docs/guides/add-an-external-idp/saml2/before-you-begin/)
```

Comment:
Should the link text be changed to match the current name of that page, 'Add an external Identity Provider', maybe adding 'SAML Authentication with OIDC' in parenthesis? Or maybe the name of the guide should itself be changed?

Reply:
Regarding 'SAML Authentication with OIDC', I think those two things are two alternatives: you'd use either SAML or OIDC, but not SAML with OIDC.

Reply:
Changed link text to "Add an External Identity Provider". 

I think a number of these guides need to be renamed, but that's a task for another day...